sporkfish/evaluator/evaluator.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC, abstractmethod
+
+
+class Evaluator(ABC):
+    @abstractmethod
+    def evaluate(self, input_data) -> float:
```

Comment:
To keep mypy happy I guess we need:
`def evaluate(self, board: Board) -> float:`